src/onegov/org/models/extensions.py

```diff
@@ -829,6 +830,7 @@ class GeneralFileLinkExtension(ContentExtension):
         id: Any
         files: relationship[list[File]]
         access: dict_property[str]
+        text: str
```

Comment:
```suggestion
```
Take advantage of `content_fields_containing_links_to_files`, it tells you what you need to process, rather than hardcoding `text`.

So do something like:
```python
for field_name in obj.content_fields_containing_links_to_files:
    if text := obj.content.get(field_name):
        obj.content[field_name] = remove_empty_links(text)
```
And extract the parsing and removal of link tags into a utility function.

Maybe you could also first do a `self[field_name].object_data != self[field_name].data` to only do this step if the user changed the contents of the field.